libs/scripts/scripts.js

```diff
@@ -38,5 +38,7 @@ const config = {
   await loadArea();
   const { default: loadModals } = await import('../blocks/modals/modals.js');
   loadModals();
+  const { getIconLibrary } = await import('../utils/decorate.js');
+  await getIconLibrary();
```

Comment:
May as well load in parallel.

```suggestion
const [{ default: loadModals }, { getIconLibrary }] = await Promise.all([
  import('../blocks/modals/modals.js'),
  import('../utils/decorate.js'),
]);
loadModals();
getIconLibrary();
```